packages/terra-status-view/src/StatusView.jsx

```diff
@@ -183,7 +183,7 @@ class StatusView extends React.Component {
     }
 
     let actionSection;
-    if (children.length) {
+    if (children.length || !Array.isArray(children)) { // Single children are objects, not arrays
```

Comment:
could we use `[React.Children.count](https://reactjs.org/docs/react-api.html#reactchildrencount) > 0` here instead?

Reply:
Good call, that's cleaner.